plugins/times-of-day/frontend/public/javascripts/countly.views.js

```diff
@@ -1,225 +1,118 @@
-                    "mRender": function(hour) {
-                        var nextHour = hour + 1 > 23 ? 0 : hour + 1;
-                        return (hour < 10 ? "0" + hour : hour) + ":00 - " + (nextHour < 10 ? "0" + nextHour : nextHour) + ":00";
-                    },
-                    "sType": "string",
-                    "sTitle": jQuery.i18n.map['times-of-day.hours']
+                title: {
+                    text: CV.i18n('times-of-day.title')
                 },
-                {
-                    "mData": "monday",
-                    "sType": "numeric",
-                    "sTitle": jQuery.i18n.map['times-of-day.monday'],
-                    "mRender": function(d) {
-                        return countlyCommon.formatNumber(d);
+                tooltip: {
+                    position: 'top',
+                    trigger: 'item',
+                    formatter: function(params) {
+                        return '<div class="bu-is-flex bu-is-flex-direction-column times-of-day__scatter-chart-tooltip"> \n' +
+                                    '<span class="times-of-day__scatter-chart-tooltip-text">' + CV.i18n('times-of-day.total-users') + '</span>\n' +
+                                    '<span class="times-of-day__scatter-chart-tooltip-total-users-value">' + params.value[2] + '</span> \n' +
+                                    '<span class="times-of-day__scatter-chart-tooltip-text">' + CV.i18n('times-of-day.between') + ' ' + countlyTimesOfDay.service.getHoursPeriod(countlyTimesOfDay.service.HOURS[params.value[0]]) + '</span> \n' +
+                                '</div>';
                     }
                 },
-                {
-                    "mData": "tuesday",
-                    "sType": "numeric",
-                    "sTitle": jQuery.i18n.map['times-of-day.tuesday'],
-                    "mRender": function(d) {
-                        return countlyCommon.formatNumber(d);
-                    }
-                },
-                {
-                    "mData": "wednesday",
-                    "sType": "numeric",
-                    "sTitle": jQuery.i18n.map['times-of-day.wednesday'],
-                    "mRender": function(d) {
-                        return countlyCommon.formatNumber(d);
-                    }
-                },
-                {
-                    "mData": "thursday",
-                    "sType": "numeric",
-                    "sTitle": jQuery.i18n.map['times-of-day.thursday'],
-                    "mRender": function(d) {
-                        return countlyCommon.formatNumber(d);
-                    }
-                },
-                {
-                    "mData": "friday",
-                    "sType": "numeric",
-                    "sTitle": jQuery.i18n.map['times-of-day.friday'],
-                    "mRender": function(d) {
-                        return countlyCommon.formatNumber(d);
+                xAxis: {
+                    type: 'category',
```

Comment:
In the BaseChart class which every chart inherits has xAxis by default 'category' btw.